src/normalizer.ts

```diff
@@ -224,8 +243,8 @@ const normalizeSlideObject = (
         color: style.color ? normalizeHexColor(style.color) : null,
         fontFace: style.fontFace ?? DEFAULT_FONT_FACE,
         fontSize: style.fontSize ?? DEFAULT_FONT_SIZE,
-        align: style.align ?? "left",
-        verticalAlign: style.verticalAlign ?? "middle",
+        align: style.align,
+        verticalAlign: style.verticalAlign,
```

Comment:
As a note, here I had to remove the default alignments because this messes with bullet points ("left" seems to merge multiple bullet points into one bullet for some reason). I didn't notice any regressions for normal text though from leaving this as undefined, unless you know of any particular reasons why this default option was passed?

Reply:
Seems good. I think the default option was set because some pptx readers (maybe OS X preview?) had some weird defaults if alignments were set to undefined